src/diffusers/pipelines/stable_diffusion/pipeline_onnx_stable_diffusion.py

```diff
@@ -197,15 +198,20 @@ def __call__(
         if accepts_eta:
             extra_step_kwargs["eta"] = eta
 
+        timestep_dtype = next(
+            (input.type for input in self.unet.model.get_inputs() if input.name == "timestep"), "tensor(float)"
+        )
+        timestep_dtype = ORT_TO_NP_TYPE[timestep_dtype]
```

Comment:
Passing float timesteps (as in K-LMS) to the onnx unet will result in a type error. This adds a bit of back-compatibility until we update checkpoints to accept `float` timesteps.

(Can't change them to `float` right away, that will break pipelines from older version of `diffusers`)